src/helpers/__tests__/utils.test.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, it, expect } from 'vitest';
+
+import { createErc1155Metadata } from '../utils';
+
+describe('utils', () => {
+  describe('createErc1155Metadata', () => {
+    it('should create metadata', () => {
+      const metadata = createErc1155Metadata({
+        name: 'Test',
+        description: 'Test description',
+        externalUrl: 'https://test.com',
+        githubUrl: 'https://github.com',
+        twitterUrl: 'https://twitter.com',
+        discordUrl: 'https://discord.com',
+        treasuryAddress: '0x000000000000000000000000000000000000dead'
+      });
+
+      expect(metadata).toEqual({
+        name: 'Test',
+        description: 'Test description',
+        external_url: 'https://test.com',
+        properties: {
+          github_url: 'https://github.com',
+          twitter_url: 'https://twitter.com',
+          discord_url: 'https://discord.com',
+          treasury_address: '0x000000000000000000000000000000000000dead'
```

Comment:
This should be an array allowing multiple wallets and we should add the network too, I would name it `wallets` like:
```suggestion
          wallets: [
            'gor:0x000000000000000000000000000000000000dead'
          ]
```
On the UI we don't need to allow input multiple wallets we can force looking at the first wallet.